metadata/routes.py

```diff
@@ -69,6 +69,24 @@ def get_credentials():
         response.status = e.status
         return {'error': {'message': e.message}}
 
+@route('/latest/dynamic/instance-identity/document')
+def get_identity_document():
+    return {
+        'privateIp' : '127.0.0.1',
+        'devpayProductCodes' : None,
+        'availabilityZone' : 'us-east-1a',
+        'version' : '2010-08-31',
+        'instanceId' : 'i-12345678',
+        'billingProducts' : None,
+        'instanceType' : 't2.nano',
+        'accountId' : '123456789012',
```

Comment:
You could get the account id using the boto session. I remember it was a bit of hassle the last time I had to do it. Something like parsing it of an error response. That was if you wanted to do it consistently, even if the iam account does not have GetUser. Anyway, probably not worth doing it here.